crates/cast/bin/cmd/send.rs

```diff
@@ -216,11 +225,11 @@ corresponds to the sender, or let foundry automatically detect it by not specify
 }
 
 #[allow(clippy::too_many_arguments)]
-async fn cast_send<M: Middleware, P: TempProvider, F: Into<NameOrAddress>, T: Into<NameOrAddress>>(
+async fn cast_send<M: Middleware, P: TempProvider>(
     provider: M,
     alloy_provider: P,
```

Comment:
why do we need both?

Reply:
We instantiate `Cast` here, which needs both providers